platform/security/handler/security-handler-oidc/src/main/java/org/codice/ddf/security/handler/oidc/OidcHandler.java

```diff
@@ -191,6 +216,13 @@ private boolean userAgentIsNotBrowser(HttpServletRequest httpRequest) {
   }
 
   private OidcCredentials getCredentialsFromRequest(J2EContext j2EContext) {
+    // Check that the request contains a code, an access token or an id token
+    Map<String, String[]> requestParams = j2EContext.getRequestParameters();
+    if (!requestParams.containsKey("code")
+        && !requestParams.containsKey("access_token")
+        && !requestParams.containsKey("id_token")) {
+      return new OidcCredentials();
+    }
```

Comment:
❗️ I believe it is possible to receive an access token in the authentication header https://auth0.com/docs/tokens/guides/access-token/use-access-tokens#custom-api-access-token-interactions.